src/OpenTelemetry.AutoInstrumentation.Native/otel_profiler_constants.h

```diff
@@ -15,6 +15,7 @@ const WSTRING env_vars_to_display[]{environment::clr_profiler_enabled,
                                     environment::integrations_path,
                                     environment::include_process_names,
                                     environment::exclude_process_names,
+                                    environment::enabled_integrations,
```

Comment:
It would be better to log all env vars having following prefixes:
- `COR_`
- `DOTNET_`
- `OTEL_`

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1325